### DISCUSSION TITLE
What is the purpose of Manual in useCloned?

### DISCUSSION THREAD

**Discussion by Zerthz** [vueuse/vueuse#3998] [Q&A] [answered]
Heyo, I am looking at the docs for useCloned. And I'm struggling to understand the purpose behind the manual keyword. If I understand correctly you can activate sync( ) to  sync the clone to the original. 

The default is that it never syncs right? Since it's a clone, just feels like that false manual sync is automatic sync not no sync.

**Comment by ilyaliao** [comment, marked as answer]
If `manual` is set to `true`, you must use `sync()` to synchronize manually; by default, synchronization is automatic.
